examples/servients/opcua-cli/package.json

```diff
@@ -18,16 +18,16 @@
   },
   "devDependencies": {
     "@types/node": "^10.9.4",
-    "ts-node": "3.3.0",
-    "typescript": "2.9.2",
+    "ts-node": "8.6.2",
+    "typescript": "3.6.3",
     "typescript-standard": "0.3.30",
     "wot-typescript-definitions": "0.7.3"
   },
   "dependencies": {
-    "@node-wot/core": "0.7.2",
-    "@node-wot/binding-http": "0.7.2",
-    "@node-wot/binding-file": "0.7.2",
-    "@node-wot/binding-opcua": "0.7.2",
+    "@node-wot/binding-file": "0.7.5",
+    "@node-wot/binding-http": "0.7.5",
+    "@node-wot/binding-opcua": "0.7.5",
```

Comment:
here it should be "0.7.6"

Reply:
ok

Reply:
> here it should be "0.7.6"

Did you mean 0.7.7 ?

Reply:
In  this section we seem to be sticking to the _released_ version that currently is 0.7.6
Maybe it is even easier to have something like `0.7.x` here